agent/src/main/java/com/jonnyzzz/teamcity/plugins/node/agent/nvm/NVM.kt

```diff
@@ -96,6 +96,7 @@ class NVMRunner(val downloader : NVMDownloader,
     val version = context.runnerParameters[bean.NVMVersion]
     val fromSource = if(!context.runnerParameters[bean.NVMSource].isEmptyOrSpaces()) "-s " else ""
     val url = context.runnerParameters[bean.NVMURL] ?: bean.NVM_Creatonix
+    var isWindows = runningBuild.agentConfiguration.systemInfo.isWindows
```

Comment:
`val`. You probably are not going to change the value of it. Mutable state is hard to maintain anyways.

Reply:
👍